docs/src/references/cli/tedge-http.md

```diff
@@ -0,0 +1,224 @@
+---
+title: "tedge http"
+tags: [Reference, CLI]
+sidebar_position: 6
+---
+
+# The tedge http command
+
+A `tedge` sub command to interact with the HTTP services hosted on the device by the Cumulocity mapper and the agent:
+
+- the [Cumulocity proxy](/references/cumulocity-proxy/)
+- the [file transfer service](/references/file-transfer-service)
+- the entity store.
```

Comment:
@albinsuresh do we have some documentation on the HTTP API for device registration?

Reply:
Yes. At `doc/src/operate/registration/register.md`

Reply:
I added the link removing the draft status of this page (a draft page cannot be linked to): https://github.com/thin-edge/thin-edge.io/pull/3357/commits/f71705349e51b269ab232631e3b7417334a342d8